src/table/dtable.jl

```diff
@@ -0,0 +1,46 @@
+import DataFrames
+import Tables
+
+import Base: fetch, filter, map, setproperty!, eachrow
+
+mutable struct DTable
+    v::Vector{Dagger.EagerThunk}
+    thunksize::Int
```

Comment:
I agree, I've added this at some point and didn't really think about it.
The chunk/partition size would only be relevant on creation and only in case the source needs to be partitioned by the constructor.
At this point I'll remove it as I don't really see a point in keeping it either